models/tasks.js

```diff
@@ -193,6 +193,50 @@ const fetchUserTasks = async (username, statuses = [], field, order) => {
   }
 };
 
+const fetchDbForTask = async (skill, level) => {
+  const task = await tasksModel
+    .where("taskLevel.category", "==", skill)
+    .where("taskLevel.level", ">=", level)
+    .where("taskLevel.level", "<=", level + 2)
+    .where("status", "==", "AVAILABLE")
+    .limit(1)
+    .get();
+  return task;
+};
+
+/**
+ *
+ * @param skill { string } : skill category which will be used
+ * @param level { number } : level of the skill
+ * @returns {Promise<task>|object}
+ */
+
+const fetchSkillLevelTasks = async (skill, level) => {
```

Comment:
Let's have singular name `fetchSkillLevelTask`.

Reply:
okay will do it